migrations/2_deploy_organization.js

```diff
@@ -30,50 +31,48 @@ module.exports = async (deployer) => {
   const orgName = "Genesis";
   const tokenName = "Gen";
   const tokenSymbol = "GEN";
-  const founders = [web3.eth.accounts[0]];
-  const initRepInWei = [web3.toWei(10)];
-  const initTokenInWei = [web3.toWei(1000)];
   const orgNativeTokenFee = 0;
   const defaultVotingMachineParams = {
     preBoostedVoteRequiredPercentage: 50,
-    preBoostedVotePeriodLimit: 60,
-    boostedVotePeriodLimit: 60,
-    thresholdConstA: 1,
-    thresholdConstB: 1,
+    preBoostedVotePeriodLimit: 5184000, // 2 months
+    boostedVotePeriodLimit: 604800, // 1 week
+    thresholdConstA: 1, // TODO: how many proposals can be boosted at a time. want about 5, have to read arc to figure out
```

Comment:
How this param is related to "how many proposals can be boosted at a time"?

Reply:
@leviadam @tibetsprague

Reply:
* @dev threshold return the organization's score threshold which required by
     * a proposal to shift to boosted state.
     * This threshold is dynamically set and it depend on the number of boosted proposal.

Reply:
ok so @fmatan suggests thresholdConstA: 2, thresholdConstB: 10, to start